openfisca_core/parameters.py

```diff
@@ -255,7 +259,7 @@ class ParameterAtInstant(object):
 
     _allowed_value_data_types = [int, float, bool, type(None)]
     # 'unit' and 'reference' are only listed here for backward compatibility
-    _allowed_keys = set(['value', 'metadata', 'unit', 'reference'])
+    _allowed_keys = COMMON_KEYS.union({'value'})
```

Comment:
@fpagnoux Is this the expected behaviour?

Reply:
No, good point, I don't think we should have `description` and `documentation` for dated values.